test.js

```diff
@@ -0,0 +1,39 @@
+[
+	{
```

Comment:
might want to do a pr that takes this file out of your git repo. You can't remove it permanently from your history (well, not without a lot of work), but you can at least remove it from the active code on the branch.